setup.py

```diff
@@ -74,7 +74,7 @@
     'invenio-oaiharvester==1.0.0a2',
     'invenio-utils==0.2.0',  # Not fully Invenio 3 ready
     'invenio>=3.0.0a1,<3.1.0',
-    'inspire-crawler~=0.2.7',
+    'inspire-crawler~=0.0,>=0.2.7',
```

Comment:
yes, but if you pair the first like this: `~=0.0,>=0.2` it's even more meaningful and less prone to errors of pinning to the feature version, say for example that you have to change it and pin it to a minimum version of 0.3.5, with the single solution it's easy to end up with `~=0.3.5`, and that will not do what you want, as it will pin to the feat version, so you will never get 0.4, but with this solution you will most probably end up with `~=0.0,>=0.3.5`, that does exactly what you want. The idea of using the 0.0 instead of 0.2/0.3 is to make clear that the first part of the version restriction is only to pin the major version.